distributed/client.py

```diff
@@ -2989,6 +2991,10 @@ def _get_computation_code(
 
         ``stacklevel`` may be used to explicitly indicate from which frame on
         the stack to get the source code.
+
+        Returns
+        -------
+        tuple[SourceCode, ...]
```

Comment:
This is fine, but also somehow seems redundant given the type annotation (also seems like another way for the docstring to become outdated). Not sure what the best practice is here -- maybe @crusaderky @graingert have thoughts?

Reply:
For (completely unenforced) reference:

> Type specifications should also be avoided in Sphinx documentation when redundant with the annotations in the code.

https://github.com/dask/community/issues/255